src/main/java/core/basesyntax/service/impl/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operations;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.DataReader;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFileReaderImpl implements DataReader {
+    private static final String DELIMITER = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final int DATA_TYPES_QUANTITY = 3;
+    private static final int LOWER_BOUND_OF_BALANCE = 0;
+
+    @Override
+    public List<Transaction> read(String filepath) {
+        try (BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(filepath))) {
+            List<Transaction> lineList = new ArrayList<>();
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                if (!line.contains("type")) {
```

Comment:
we can remove this check by just skipping the first line when reading

Reply:
Fixed